src/rust/rust-proto/src/graplinc/grapl/api/graph_query/v1beta1/messages.rs

```diff
@@ -42,6 +45,15 @@ impl From<QueryId> for proto::QueryId {
     }
 }
 
+impl type_url::TypeUrl for QueryId {
+    const TYPE_URL: &'static str =
+        "graplsecurity.com/graplinc.grapl.api.graph_query_service.v1beta1.QueryId";
```

Comment:
!!! This may have changed since you started this PR, but I renamed these to `graph_query` to remove this extraneous `_service`. 

https://github.com/grapl-security/grapl/pull/2032